README.md

```diff
@@ -61,7 +77,10 @@ mirrord exec node app.js --pod-name my-pod
 ---
 
 ## How It Works
-mirrord works by letting you select a pod to mirror traffic from. It launches a privileged pod on the same node which enters the namespace of the selected pod and captures traffic from it.
+mirrord works by letting you select a pod to plug into (mirror incoming traffic from, send outgoing traffic through, read and write files to, and read environment variables from). It launches a privileged pod on the same node which enters the namespace of the selected pod.
```

Comment:
Hmm I would suggest moving the features inside `()` to the end.

```suggestion
mirrord works by letting you select a pod to plug into. It launches a privileged pod on the same node which enters the namespace of the selected pod, and enables you to  mirror incoming traffic from, send outgoing traffic through, read and write files to, and read environment variables from.
```

Something like this, with a better (read proper) "connectivity" rather than just this crappy "and enables you to".

Reply:
Rewrote it, what do you think?